ext/pdo_firebird/tests/pdofirebird_001.phpt

```diff
@@ -0,0 +1,44 @@
+--TEST--
+PDO_firebird subclass basic
+--EXTENSIONS--
+pdo_firebird
+--SKIPIF--
+<?php require(__DIR__ . '/skipif.inc'); ?>
+--FILE--
```

Comment:
@kocsismate See https://github.com/php/php-src/pull/12699, these should be marked with `XLEAK`. For reference: https://github.com/php/php-src/actions/runs/7509344302/job/20446349499

Reply:
@iluuu1994 
ah! I knew about the firebird leak, but I overlooked it in the review.

@kocsismate
This and 002 also require XLEAK.